src/holo-crypto/crates/holo-crypto-wasm/src/lib.rs

```diff
@@ -0,0 +1,51 @@
+mod utils;
+
+use wasm_bindgen::prelude::*;
+
+use holo_crypto::{
+	error::HoloCryptoError,
+	KeyPair,
+};
+
+// When the `wee_alloc` feature is enabled, use `wee_alloc` as the global
+// allocator.
+#[cfg(feature = "wee_alloc")]
+#[global_allocator]
+static ALLOC: wee_alloc::WeeAlloc = wee_alloc::WeeAlloc::INIT;
+
+#[wasm_bindgen]
+pub struct AdminKeyPair(KeyPair);
+
+#[wasm_bindgen]
+impl AdminKeyPair {
+    #[wasm_bindgen(constructor)]
+	pub fn new(email: &str, pass: &str, hc_pub_key: &str) -> Result<AdminKeyPair, JsValue> {
+		let combined_passphrase = pass.to_owned() + email;
```

Comment:
I was not sure how we want to handle the email, password and hc_pub_key. For the first pass I am concatenating the password and email and using the hc_pub_key as salt. This probably needs some more consideration